tests/behavior/presign.rs

```diff
@@ -134,15 +134,10 @@ pub async fn test_presign_stat(op: Operator) -> Result<()> {
     }
     let resp = req.send().await.expect("send request must succeed");
     assert_eq!(resp.status(), http::StatusCode::OK, "status ok",);
-    // response headers default content_length method cannot get the correct value
-    let content_length = resp
-        .headers()
-        .get(header::CONTENT_LENGTH)
-        .expect("content length must exist")
-        .to_str()
-        .expect("content length must be valid str")
-        .parse::<u64>()
-        .expect("content length must be valid u64");
+
+    let content_length = raw::parse_content_length(resp.headers())
+        .expect("no content length")
+        .expect("content length must be present");
     assert_eq!(content_length, size as u64);
```

Comment:
We can use `assert_eq!(content_length, Ok(Some(size as u64)))` to make the code bit better for reading.